pkg/ebpfev/event_interface.go

```diff
@@ -0,0 +1,14 @@
+package ebpfev
+
+import "time"
+
+type EventData struct {
```

Comment:
[Question]: What interface is covering this struct? Where are the getters and setters?

Reply:
meanwhile I don't think I need to use it has an interface, this is a place holder If I'll need in the future, too many getters and setters! I prefer to use them as public

Reply:
You should write getters and setters for a struct to provide encapsulation, which makes it possible to hide the internal state of an object and control access to its properties.

Getters are used to retrieve the value of a struct property, while setters are used to change the value of a struct property. These methods allow you to enforce rules and constraints on how a struct's properties can be modified and accessed, making it easier to maintain the integrity of your data and prevent bugs.

Using an interface to define the getter and setter methods can be useful in some cases, as it allows you to define a common contract that multiple structs can implement, making it easier to write generic code that works with multiple types.

Reply:
done